src/main/java/com/microsoft/lst_bench/util/StringUtils.java

```diff
@@ -50,6 +50,13 @@ public static String quote(String str) {
     return "'" + str + "'";
   }
 
+  public static String replaceSingleQuotes(String str) {
+    if (str == null) {
+      return null;
+    }
+    return str.replace("'", "\"");
```

Comment:
It would be better to replace by `''`, which is the usual way of escaping a quote in SQL.